pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>id.shivelight.paper</groupId>
     <artifactId>unexpectedspawn</artifactId>
-    <version>0.2.1</version>
+    <version>0.2.2</version>
```

Comment:
This is huge and breaking changes, bump the version to 0.3.0.

Reply:
Will update the version once I am back on PC. Currently on phone.